docs/runtime-contract.md

```diff
@@ -394,15 +394,15 @@ invalid, the container execution MUST be failed.
 
 ### Default Filesystems
 
-The OCI specification describes a default container environment which may be
+The OCI specification describes a default container environment which might be
 used for many different purposes, including containerization of existing legacy
-or stateful processes which may store substantial amounts of on-disk state. In a
+or stateful processes which might store substantial amounts of on-disk state. In a
 scaled-out, stateless environment, container startup and teardown is accelerated
-when on-disk resources are kept to a minimum. Additionally, developers may not
-have access to the container filesystems (or the containers may be rapidly
+when on-disk resources are kept to a minimum. Additionally, developers might not
+have access to the container filesystems (or the containers might be rapidly
 recycled), so log aggregation SHOULD be provided.
 
-In addition to the filesystems recommended in the OCI, the following filesystems
+In addition to the filesystems suggested in the OCI, the following filesystems
```

Comment:
I think this is RECOMMENDED if it's coming from OCI.

Reply:
+1

Reply:
OCI might recommend them, but I'm not sure the use of the word here is normative since we're not actually adding a new constraint we're just saying "they tell us to do this".  But let me think about it....

Reply:
IMO it would be great to add the OCI recommended filesystems inline with a link to where they come from. No one is going to click out to find what are the recommended filesystems.

Reply:
Ha! The OCI spec is actually SHOULD:

https://github.com/opencontainers/runtime-spec/blob/master/config-linux.md#default-filesystems